AspNetCoreTodo/Controllers/TodoController.cs

```diff
@@ -26,31 +25,49 @@ public class TodoController : Controller
         /// <param name="userManager"></param>
         public TodoController(  ITodoItemService todoItemService,
                                 UserManager< ApplicationUser> userManager)
+            :base (userManager)
         {
             _todoItemService = todoItemService;
-            _userManager = userManager;
         }
 
 
 
+        public async Task<IActionResult> Index(/*bool completedItems = false*/)
+        {
+            //obtencion del usuario logeado
+            var currentUser = await _userManager.GetUserAsync( User);
+            if (currentUser == null)
+                return Challenge(); //fuerza el logeo, mostrando la pagina de logeo
+
+            var items = await _todoItemService.GetIncompletItemsAsync( currentUser);
 
+                //Current user se hace null para poder ver todos los Todo's.
+                currentUser = null;
+            }
```

Comment:
```suggestion
            }
```
Está haciendo que no compile la solución